datasets/sentinel-3/collection/sentinel-3-synergy-vgp-l2-netcdf/template.json

```diff
@@ -0,0 +1,281 @@
+{
+    "stac_version": "1.0.0",
+    "type": "Collection",
+    "id": "sentinel-3-synergy-vgp-l2-netcdf",
+    "title": "Sentinel-3 Top of Atmosphere Reflectance",
```

Comment:
Should this title (and the `msft:short_description`) mention something about vegetation?

Reply:
Yeah, this is a little muddy. The VGP product (along with V10 and VG1) is a SPOT Continuity Product. Specifically, it provides continuity for the VEGETATION instrument that flew on SPOT-3 and SPOT-4. So even though this collection only contains reflectance data, the reflectance data mirrors that from a prior sensor that was focused on vegetation.

- I added `(SPOT VEGETATION Continuity)` to the template `title`
- I appended `, a SPOT VEGETATION continuity product` to the template `msft:short_description`
- The longs description also mentions the fact that this is a SPOT VEGETATION instrument continuity product.

These same changes were applied to the V10 and VG1 collections.